codegen/smithy-dafny-codegen-test/src/test/java/software/amazon/polymorph/smithyrust/RustTestModels.java

```diff
@@ -13,8 +13,12 @@
 
 class RustTestModels extends TestModelTest {
 
+  // Tests in this set will never be run
   private static final Set<String> DISABLED_TESTS = new HashSet<>();
 
+  // If this set is nonempty, only included tests will be run
+  private static final Set<String> ENABLED_TESTS = new HashSet<>();
```

Comment:
Nice! I have an alternative version that accepts an environment variable filter pattern. Do you think that would be useful too?

Reply:
Yes, that would work just as well. If you wanted to merge that to main-1.x first then I can merge it into this PR, or you could push directly to this branch, whichever's easier.